data/Fizzics/objectEditor.ui

```diff
@@ -9,8 +9,8 @@
     <property name="page_increment">10</property>
   </object>
   <object class="GtkAdjustment" id="adjustmentFriction">
-    <property name="upper">18</property>
-    <property name="value">9</property>
+    <property name="upper">100</property>
+    <property name="value">5</property>
     <property name="step_increment">1</property>
```

Comment:
Bah, only noticed this when tested today:

We need to increase the `step_increment` from 1 to 10 here and change the `value` from 5 to 10 instead. Two reasons 1) now that the value goes to 100, moving it 1 by 1 takes forever and  2) that is exactly what we already do with a similar one (radius).

Reply:
Being able to easily set numbers in the 0-10 range is important because how different it is. I'd be afraid that making 5 the default increment would make it harder to fine tune them. Besides, they can type any number if they want to get very larger, right?

Reply:
The range of values goes from 0 to 100, the UI should allow them to move quickly between the whole range. If they need finer tuning, like you just said, the user can type any number between that range.

Reply:
I disagree with this, but I'm making the change anyway because I think it's more important to get this PR in than to wait for another round of discussions.